testing/adios2/install/CXX11/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
+
+cmake_minimum_required(VERSION 3.6)
+project(adios_c_test CXX)
```

Comment:
Should `project(adios_c_test CXX)` be `project(adios_cxx11_test CXX)`?

Reply:
Indeed, good catch